src/renderer/ducks/beacon/sagas.ts

```diff
@@ -222,17 +206,20 @@ export function* watchOnHead(
                 }
                 continue;
             }
-            if (payload.value.type === BeaconEventType.HEAD) {
-                yield put(updateSlot(payload.value.message.slot, url));
-                if (isSyncing || !isOnline) {
-                    const result = yield client.node.getSyncingStatus();
-                    isSyncing = result.syncDistance > 10;
-                    isOnline = true;
-                    yield put(updateStatus(isSyncing ? BeaconStatus.syncing : BeaconStatus.active, url));
-                }
+
+            yield put(updateSlot(payload.value.message.slot, url));
+            const headEpoch = Math.floor(payload.value.message.slot / 32);
```

Comment:
there is method for that in state transition package:
https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar-beacon-state-transition/src/util/epoch.ts#L13